CMakeLists.txt

```diff
@@ -131,7 +142,7 @@ if(${BLAS_ENABLE_BENCHMARK})
 endif()
 
 option(BLAS_ENABLE_AUTO_TUNERS "Whether to enable building GEMM auto tuners" OFF)
-if(${BUILD_AUTO_TUNERS})
+if(${BUILD_ENABLE_AUTO_TUNERS})
```

Comment:
Should this not be $BLAS_ENABLE_AUTO_TUNERS instead of $BUILD_ENABLE_AUTO_TUNERS? I can't see any option or variable with that name unless it is coming from somewhere else.

Reply:
Yes, well spotted.